src/components/reusable/SearchBar.js

```diff
@@ -57,30 +46,39 @@ const Searchbar = ({ selectItem, getSearchResults }) => {
 
   // Keyboard input in search box
   const handleTextChange = (event) => {
-    // When new text is typed the dropdown should be closed
     closeDropdown();
-    event.preventDefault();
-    // Update the search value to the new text
     setSearchValue(event.target.value);
+
+    // If an index has been highlighted, should navigate on enter
+    if (highlightedIndex >= 0) {
```

Comment:
This shouldnt be here. Selecting an item should never be done on text change. This will make it so if anything is selected and you type a new letter then you will naviage to the other page. The only thing that should happen on text change is getting more search results.

Reply:
done